boards/posix/nrf52_bsim/Kconfig

```diff
@@ -0,0 +1,32 @@
+
+if BOARD_NRF52_BSIM
+
+comment "NRF52_BSIM options"
+
+config PRINTK_HOOK_INIT_PRIORITY
+	int
+	depends on BOARD_NRF52_BSIM
+	default 50
+	help
+	  Just the driver init priority
+
+endif # BOARD_NRF52_BSIM
+
+
+#This would eventually be shared by a possible family of simulated NRF boards
+#which use BabbleSim. When that happens, we can move this to a common
+#Kconfig file
+
+config SOC_SERIES_BSIM_NRFXX
+	bool
+	default n
```

Comment:
`bool` symbols implicitly default to `n`, so `default n` can be skipped.

That generates clearer docs too:

    Implicitly defaults to n

instead of

    - n if <dependencies>